site/docs/concepts/collections.md

```diff
@@ -15,6 +15,39 @@ you define one or more new collections as part of that process.
 Every collection has a key and an associated [schema](#schemas)
 that its documents must validate against.
 
+## Viewing collection data
+
+In many cases, it's not necessary to view your collection data — you're able to materialize it directly to a destination in the correct shape using a [connector](../concepts/README.md#connectors).
+
+However, it can be helpful to view collection data to confirm the source data was captured as expected, or verify a schema change.
+
+#### In the web application
+
+Sign into the Flow web application and click the **Collections** tab. The collections to which you have access are listed.
+Click the **Details** drop down to show a sample of collection data as well as the collection [specification](#specification).
+
+The collection documents are displayed by key. Click the desired key to preview it in its native JSON format.
+
+#### Using the flowctl CLI
+
+In your [authenticated flowctl session](../reference/authentication.md#authenticating-flow-using-the-cli), issue the command `flowctl collections read --collection <full/collection-name> --uncommitted`. For example, `flowctl collections read --collection acmeCo/inventory/anvils --uncommitted`.
+
+Options are available to read a subset of data from collections.
+For example, `--since` allows you to specify an approximate start time from which to read data, and
+`--include-partition` allows you to read only data from a specified [logical partition](../concepts/advanced/projections.md#logical-partitions).
+Use `flowctl collections read --help` to see documentation for all options.
+
+:::info Beta
+While in beta, this command currently has the following limitations. They will be removed in a later release:
+
+* The `--uncommitted` flag is required. This means that all collection documents are read, regardless of whether they were successfully committed or if they part of a [transaction](../concepts/advanced/shards.md#transactions) that was rolled back or uncommitted.
```

Comment:
> or if they part of a [transaction](../concepts/advanced/shards.md#transactions) that was rolled back or uncommitted.

This just seems like a kind of awkward ending, and can be replace with "or not."